modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -127,6 +129,30 @@
 (defn- num-to-ds-interval [unit v] (hsql/call :numtodsinterval v (hx/literal unit)))
 (defn- num-to-ym-interval [unit v] (hsql/call :numtoyminterval v (hx/literal unit)))
 
+(def ^:private legacy-max-identifier-length
+  "Maximal identifier length for Oracle < 12.2"
+  30)
+
+(defn- truncate-identifier
+  [identifier]
+  (->> identifier
+       hash
+       str
+       (map (fn [digit]
+              (-> digit
+                  int
+                  (+ 65)
```

Comment:
where does the 65 come from?

Reply:
65 is ASCII `A` so I'm guess Simon is converting the digits in the truncated identifier here to letters

Reply:
yeah, realized after ;-)